### PR TITLE
Update feature stage

### PR DESCRIPTION
**Pull request by bianpengyuan**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by bianpengyuan** [comment]
@nrjpoddar @howardjohn @sdake @ostromart Please let me know if you want to update any feature stages. Thanks!

**Review by nmittler — APPROVED** [review]

**Comment by nrjpoddar** [comment]
@bianpengyuan can you add "Helm based install" is alpha with link to the docs? Thanks!

File: content/en/about/feature-stages/index.md [hunk 1]
```diff
@@ -82,13 +82,10 @@ Below is our list of existing features and their current phases. This informatio
 | [Standalone Operator](/docs/setup/install/operator/) | Beta
 | [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/) | Stable
 | [Kubernetes: Istio Control Plane Installation](/docs/setup/) | Stable
-| [Attribute Expression Language](https://istio.io/v1.6/docs/reference/config/policy-and-telemetry/expression-language/) | Stable
 | Mixer Out-of-Process Adapter Authoring Model | deprecated
-| [Multicluster Mesh](/docs/setup/install/multicluster/) | Alpha
-| [Kubernetes: Istio Control Plane Upgrade](/docs/setup/) | Beta
-| Consul Integration | Alpha
+| [Multicluster Mesh](/docs/setup/install/multicluster/) | Beta
```
Comment (sdake) [inline content/en/about/feature-stages/index.md:86]: @nmittler did this make beta? I thought we were proceeding - unfortunately - with alpha for this feature.

**Review by sdake — COMMENTED** [review]
helm based install is needed in here somewhere. I am not sure if the feature status is experimental or alpha. Some docs seem to indicate alpha, and I think Helm may meet the alpha requirements, although there must be an experimental and alpha checklist in the https://github.com/istio/enhancements repository.

I would recommend submitting each feature stage update as a separate PR so that one feature does not delay another feature on process.

@nmittler please confirm that muticluster has met the beta requirements.

The rest looks good. I am glad we are finally removing Consul 👍

**Comment by nrjpoddar** [comment]
@bianpengyuan here's the [Helm Alpha status enhancement issue](https://github.com/istio/enhancements/issues/12) for reference.

File: content/en/about/feature-stages/index.md (hunk 1 above)
Comment (linsun) [reply content/en/about/feature-stages/index.md:86]: Yes it did to my best knowledge

Comment (linsun) [inline content/en/about/feature-stages/index.md:86]:
```suggestion
| [Multicluster Mesh](/docs/setup/install/multicluster/) | Beta
| [External Control Plane](/docs/setup/additional-setup/external-controlplane/) | Alpha
```

Comment (linsun) [reply content/en/about/feature-stages/index.md:86]: adding External CP here cc @bianpengyuan @sdake

Comment (bianpengyuan) [reply content/en/about/feature-stages/index.md:86]: Ack thanks!

**Review by ericvn — COMMENTED** [review]
Should the 3 deprecated `mixer` entries be removed as well?

**Comment by bianpengyuan** [comment]
@ericvn removed. thanks!

File: content/en/about/feature-stages/index.md [hunk 2]
```diff
@@ -82,19 +82,15 @@ Below is our list of existing features and their current phases. This informatio
...
+| [Kubernetes: Istio Control Plane Upgrade](/docs/setup/upgrade/) | Beta
 | Basic Configuration Resource Validation | Beta
-| Configuration Processing with Galley | Beta
-| [Custom Mixer Build Model](https://github.com/istio/istio/wiki/Mixer-Compiled-In-Adapter-Dev-Guide) | deprecated
-| [Out of Process Mixer Adapters (gRPC Adapters)](https://github.com/istio/istio/wiki/Mixer-Out-Of-Process-Adapter-Dev-Guide) | deprecated
 | [Istio CNI plugin](/docs/setup/additional-setup/cni/) | Alpha
 | IPv6 support for Kubernetes | Alpha. Dual-stack IPv4 and IPv6 is not supported.
 | [Distroless base images for Istio](/docs/ops/configuration/security/harden-docker-images/) | Alpha
 | [Virtual Machine Integration](/docs/setup/install/virtual-machine/) | Alpha
+| [Install with Helm](/docs/setup/install/helm/) | Alpha
```
Comment (bianpengyuan) [inline content/en/about/feature-stages/index.md:93]: @nrjpoddar Does the wording here look good to you?

Comment (bianpengyuan) [reply content/en/about/feature-stages/index.md]: Updated to `Helm Based Installation`

**Review by nrjpoddar — COMMENTED** [review]
Approving for Helm feature status.

**Comment by bianpengyuan** [comment]
@ericvn please take a look, thanks!

**Review by ericvn — APPROVED** [review]

**Pull request merged**